config/initializers/bugsnag.rb

```diff
@@ -1,5 +1,5 @@
-if Rails.env.production? && ENV['BUGSNAG_API_KEY'].present?
+if ENV['BUGSNAG_API_KEY'].present?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.